### PR TITLE
fix(protocol-designer): remove opentrons phrase from checkboxes in select pipette screen

### PR DESCRIPTION
**Pull request by koji**
# Overview
move the function from editinstrumentsmodal to util and add 2 new words

[related thread 1](https://opentrons.slack.com/archives/C07E9T2NAQK/p1728071612776299)
[related thread 2](https://opentrons.slack.com/archives/C07E9T2NAQK/p1728589771588409)

close RQA-3307

## Risk assessment
low

### CODE REVIEW

File: protocol-designer/src/utils/index.ts [hunk 1]
```diff
@@ -248,3 +248,34 @@ export function getMatchingTipLiquidSpecs(
...
+ * This function removes the following phrases from the input string:
+ * - 'Opentrons Flex 96'
+ * - 'Opentrons OT-2 96'
+ * - '(Retired)'
+ * - '96' (only if it is not the first two characters)
+ *
+ * @param {string} input - The input string from which phrases will be removed.
+ * @returns {string} - The modified string with specified phrases removed.
+ */
+export const removeOpentronsPhrases = (input: string): string => {
```
Comment (ncdiehl11) [inline protocol-designer/src/utils/index.ts:264]: nice

File: protocol-designer/src/utils/index.ts [hunk 2]
```diff
@@ -248,3 +248,34 @@ export function getMatchingTipLiquidSpecs(
...
+  const phrasesToRemove = [
+    'Opentrons Flex 96',
+    'Opentrons OT-2 96',
+    '\\(Retired\\)',
+    '96',
+  ]
+
+  let updatedText = phrasesToRemove
```
Comment (ncdiehl11) [inline protocol-designer/src/utils/index.ts]: nit, any reason not to assign to const and chain `replace` at the end?
```suggestion
  let updatedText = phrasesToRemove.reduce((text, phrase) => {
      return text.replace(new RegExp(phrase, 'gi'), '')
    }, input)
    .trim().replace(/\s+/g, ' ').trim()
```

**Review by ncdiehl11 — APPROVED** [review]
tested and works well, left a style question

File: protocol-designer/src/utils/index.ts (hunk 2 above)
Comment (koji) [reply protocol-designer/src/utils/index.ts]: good catch, thanks!

**Pull request merged**